document-store/src/test/java/org/hypertrace/core/documentstore/postgres/query/v1/PostgresQueryParserTest.java

```diff
@@ -41,7 +42,7 @@ void testParseSimpleFilter() {
     String sql = postgresQueryParser.parse(query);
     Assertions.assertEquals(
         "SELECT * FROM testCollection "
-            + "WHERE document->'quantity' IS NULL OR CAST (document->>'quantity' AS NUMERIC) != ?",
+            + "WHERE document->'quantity' IS NULL OR CAST (document->'quantity' AS NUMERIC) != ?",
```

Comment:
See the comments above for all the test case related changes in this change - https://github.com/hypertrace/document-store/pull/95/files#r917602631